test/mocha/10-restrictions.js

```diff
@@ -340,10 +374,11 @@ describe('restrictions', function() {
     should.exist(actualRestriction.restriction);
     actualRestriction.restriction.should.deep.equal(expectedRestriction);
 
-    const getRestriction = await restrictions.get({
+    const getRestrictions = await restrictions.getAll({
       zone: ZONES.ONE,
       resource: RESOURCES.MANGO
     });
+    const getRestriction = getRestrictions.restrictions[0];
```

Comment:
Can you assert on the length of `restrictions`?

please rename this variable.  Maybe something like:
```suggestion
    const [restrictionRecord] = getRestrictions.restrictions;
```